src/components/dialogs/filters-contingency-dialog.js

```diff
@@ -69,7 +69,7 @@ const FiltersContingencyDialog = ({
     const [description, setDescription] = useState('');
 
     const handleClose = () => {
-        onClose();
+        handleCancel();
     };
 
     const handleCancel = () => {
```

Comment:
In `handleCancel` I think you should call `onClose();` first then reset `NewFiltersContingency` and `BtnSaveListDisabled` to avoid graphic change when closing dialog

Reply:
it doesn't fix this little issue.